### DISCUSSION TITLE
Authorisation error

### DISCUSSION THREAD

**Discussion by Cotiso-QQ** [ptarmiganlabs/ctrl-q#95] [Q&A] [answered]
Hi Goran,

See attached the error encountered.
![image](https://user-images.githubusercontent.com/13799968/195901787-c05b26f1-2507-46e3-a27e-5f27c363a203.png)

I am running CtrlQ from a Windows machine that is not in the AD. 
Could this be the cause ?

**Comment by mountaindude** [comment, marked as answer]
Looks like you used -app-id instead of --app-id.

When using the long version of options they always begin with double dash.
Short version of options (when available) begin with single dash.

You also need to specify where to find the cert files, unless you have them
in a subfolder named "cert".

Use the --help option on any command to get full details about all options
for that particular command.
